soc/nordic/nrf53/sync_rtc.c

```diff
@@ -107,13 +107,14 @@ static void ppi_rtc_to_ipc(union rtc_sync_channels channels, bool setup)
 /* Free DPPI and RTC channels */
 static void free_resources(union rtc_sync_channels channels)
 {
+	nrfx_dppi_t dppi = NRFX_DPPI_INSTANCE(0);
```

Comment:
This approach was suggested here: https://github.com/nrfconnect/sdk-nrf/pull/18073#discussion_r1810366143

Reply:
I don't quite get how that justification given in the above comment applies to this case, but my point is that `static const` would cause that this structure ends up in flash, so this would give some savings on its initialization that now needs to be performed at runtime and twice in this file. But it's just a detail. I'm not going to insist on changing this.